src/data/feed/quandl.js

```diff
@@ -0,0 +1,146 @@
+import d3 from 'd3';
+
+//  https://www.quandl.com/docs/api#datasets
+export default function() {
+
+    var _columnNameMap = function(colName) {
+        colName = colName.replace(/[\s()-.]/g, '');
```

Comment:
I would leave this to the consumer as well, it might be that removing these characters leads to duplicate keys